src/routes/tools/_common/_constants.js

```diff
@@ -0,0 +1,120 @@
+import models from "~/helpers/climate-models";
+import scenarios from "~/helpers/climate-scenarios";
+import boundaries from "~/helpers/mapbox-layers";
+
+export const DEFAULT_LOCATION = {
+  id: "37907",
+  title: "240 32nd Street, Sacramento, California 95816",
+  geometry: {
+    type: "Polygon",
+    coordinates: [
+      [
+        [-121.5, 38.625],
+        [-121.4375, 38.625],
+        [-121.4375, 38.5625],
+        [-121.5, 38.5625],
+        [-121.5, 38.625],
+      ],
+    ],
+  },
+  center: [-121.4688, 38.5938],
+  bbox: [-121.5, 38.5625, -121.4375, 38.625],
+};
+
+export const DEFAULT_SCENARIOS = scenarios.filter((d) =>
+  ["rcp45", "rcp85"].includes(d.id)
+);
+
+export const DEFAULT_BOUNDARIES = boundaries
+  .filter((d) => d.metadata.group === "Boundaries")
+  .map((d) => {
+    let label;
+    if (d.id === "locagrid") {
+      label = "None";
+    } else {
+      label = d.metadata.title;
+    }
+    return { ...d, label };
```

Comment:
small suggestion, feel free to take it or leave it, but you can simplify code like this by doing:

```js
.map((d) => ({
  ...d,
  label: d.id === "locagrid" ? "None" : d.metadata.title
}))
```

Reply:
Fixed via https://github.com/berkeley-gif/caladapt-website-2021/commit/6fd5a83daa36f1e82e2eafba9e7dc285fad0718f